iDiff/README.md

```diff
@@ -0,0 +1,228 @@
+# iDiff
+
+[![Build
+Status](https://travis-ci.org/GoogleCloudPlatform/runtimes-common.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtimes-common)
+[![codecov](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common)
+
+## What is iDiff?
+
+iDiff is an image differ command line tool.  iDiff can diff two images along several different criteria, currently including:
+- Docker Image History
+- Image file system
+- apt-get installed packages
+- pip installed packages
+- npm installed packages
+
+This tool can help you as a developer better understand what is changing within your images and better understand what your images contain.
+
+## Installation
+
+TODO: add how to install
```

Comment:
You can look at how we instruct people to install minikube on each platform here:
https://github.com/kubernetes/minikube/releases